test/unit/sessions/session-manager.test.ts

```diff
@@ -401,8 +401,8 @@ describe('onPropose', () => {
 
     const sessionInfo = createPendingSession();
     const existingSession = createPendingSession();
+    sessionManager.pendingSessions = [existingSession];
```

Comment:
Instead of mocking a return value in these tests, I updated them so that it will actually fire `getPendingSession` and compare against our current `pendingSessions`.